UI.py

```diff
@@ -237,26 +239,26 @@ def CreateGetLeavesAcrossRangeOptions():
 
     return member_options
 
-def CreateMultipleLeavesEmbed(leaves, include_reason):
+def CreateLeavesAcrossRangeEmbed (leaves, include_reason):
```

Comment:
```suggestion
def CreateLeavesAcrossRangeEmbed(leaves, include_reason):
```

Don't rush your work please.

Reply:
100% correct, just want to finish all before leaving, wont happen again